### ISSUE TITLE
Make quarterly messages work for TTS Slack org

### ISSUE THREAD

**Issue by jessieay** [18F/dolores-landingham-slack-bot#234]
Last week, 18F's Slack org was moved from being for 18F only to being for all of TTS.

While our Slack now includes more people, our quarterly messages are sometimes meant to be for 18F team members only. 

Quarterly Messages are messages that will be Slack DM'd to every employee that is in the Dolores database on the first day of each quarter.

The original issue for this feature is here: https://github.com/18F/dolores-landingham-slack-bot/issues/180

We regularly import all Slack users in our Slack org into the Dolores database. As far as I know, there is currently no way to know if a Slack user is from 18F or not. So our nightly import will also include non-18F TTS employees who are in Slack.

Before the ops team creates any quarterly messages, we need to decide how to move forward.

Ideas for how to do this:
- use 18F's [team API](https://team-api.18f.gov/public/api/team/) to cross-check which Slack users are from 18F (not sure if the Team API is up to date)
- Manually mark employees in the Dolores DB as "18F employees" (or maybe mark people as NOT 18F employees, since right now we have fewer non-18F people than 18F people in our Slack org)
- Think of some other smart way to do this :)

Related question: should we disable the ability to even create a quarterly message until this is resolved?

**Comment by jmhooper** [comment]
I saw some murmurings about creating an 18F channel for news/announcements specific to 18F. If that happens maybe we could import users who are subscribed to that channel?

**Comment by joshuabailes** [comment]
I like the @jmhooper approach. Anyone know if that would work in Slack?

**Comment by jessieay** [comment]
good idea @jmhooper -- want to follow up on these murmurings and see what you find? I have been lurking in `#wg-slack` but nothing about an 18F-only team yet...
